ui/src/model-manager/topbar/InstallMissingModelsButton.tsx

```diff
@@ -22,9 +25,9 @@ interface NodeError {
 }
 
 export default function InstallMissingModelsButton({}: Props) {
-  const [missngCount, setMissingCount] = useState(0);
+  const [showMyModels, setShowMyModels] = useState(false);
+  const [missingModels, setMissingModels] = useState<MissingModel[]>([]);
   useEffect(() => {
-    const validateInput = async (input: string) => {};
     // monkey patch queue prompt api to catch errors
     const queuePrompt = app.queuePrompt as Function;
     app.queuePrompt = async function () {
```

Comment:
i wonder if we can find a better way to implement this, this feels very hacky way to get notified when the queue prompt returns error, but i can't find any better solution when writing this